migrations/20210122-add_column_latestVersion_to_commands.js

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable new-cap */
+
+'use strict';
+
+const prefix = process.env.DATASTORE_SEQUELIZE_PREFIX || '';
+const table = `${prefix}commands`;
+
+module.exports = {
+    up: async (queryInterface, Sequelize) => {
+        await queryInterface.sequelize.transaction(async (transaction) => {
```

Comment:
then they will have to manually add it in their db?

Reply:
so, this will be a breaking change. 

better to highlight - https://docs.screwdriver.cd/about/contributing/index - `For a breaking change: feat(scm): Support new scm plugin. BREAKING CHANGE: github no longer works`

@jithine @tkyi?

Reply:
agree, should highlight breaking change

Reply:
sounds good, 
@klu909  take care of it when merging or update commit message now